common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1421,6 +1421,17 @@ special_genes = {
 				boy = male
```

Comment:
Noticed shoulders and clothes clip.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127568886-dc69d964-0d6f-49dd-92d3-3367c8ae91e1.png)

</details>